modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1199,4 +1199,35 @@ definition {
 		ObjectAdmin.assertRequiredErrorPresent();
 	}
 
+	@description = "LPS-171802- Verify if the Condition card is hidden when using the trigger On Subscription Status Update"
+	@priority = "5"
+	test VerifyTheConditionCardAreHidden {
+		property portal.acceptance = "true";
+
+		task ("Given a object system") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Object",
+				panel = "Control Panel",
+				portlet = "Objects");
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Order");
+		}
+
+		task ("When an Action using the trigger On Subscription Status Update is created") {
+			ObjectAdmin.goToFieldsTab();
```

Comment:
I think I missed this the first time. Are you able to go straight to the Actions Tab? We can skip this step right?